contrib/ci/Jenkinsfile

```diff
@@ -0,0 +1,27 @@
+pipeline {
+    agent { label 'gluon-docker' }
+    environment {
+        GLUON_SITEDIR = "contrib/ci/minimal-site"
```

Comment:
Is minimal the right term? Wouldn it ultimately be what we deem necessary to test and not necessarily minimal?. 

How about just `contrib/ci/site`.

Reply:
I think I thought it is a good idea to have some prefix if we want to add more test cases later. Something like babel or specific features enabled or so. In the end, this current site is really minimal.